src/components/Compiler/Compiler.ts

```diff
@@ -96,7 +100,10 @@ export class Compiler extends BaseCompiler {
         if (!fs.existsSync(avroRecordPath)) {
             fs.writeFileSync(
                 avroRecordPath,
-                "export { BaseAvroRecord } from \"@degordian/avro-to-typescript\";\n",
+                [
+                    "export { BaseAvroRecord } from \"@degordian/avro-to-typescript\";",
+                    // ...this.imports,
+                ].join("\n"),
```

Comment:
1. no comment abuse
2. this doesn't even add `\n` to the end when there is only one element does it?

Reply:
was experimenting with if we export the logicalType from here and use relative modules in all the other places, but let to more complicated code. will remove this